src/code/z_player_lib.c

```diff
@@ -599,11 +618,17 @@ u8 sEyeMouthIndexes[][2] = {
     { 7, 2 }, { 0, 2 }, { 3, 0 }, { 4, 0 }, { 2, 2 }, { 1, 1 }, { 0, 2 }, { 0, 0 },
 };
 
-u8* sEyeTextures[] = {
+/**
+ * Link's eye and mouth textures are placed at the exact same place in adult and child Link's respective object files.
+ * This allows the array to only contain the symbols for one file and have it apply to both. This is a problem for
+ * shiftability, and changes will need to be made in the code to account for this in a modding scenario. The symbols
+ * from adult Link's object are used here.
+ */
+u64* sEyeTextures[] = {
     0x06000000, 0x06000800, 0x06001000, 0x06001800, 0x06002000, 0x06002800, 0x06003000, 0x06003800,
 };
```

Comment:
We are in the process of changing the type of textures arrays from `u64*` to `void*`

```suggestion
static void* sEyeTextures[] = {
    0x06000000, 0x06000800, 0x06001000, 0x06001800, 0x06002000, 0x06002800, 0x06003000, 0x06003800,
};
```